legal-api/tests/unit/services/filings/validations/test_correction_firms.py

```diff
@@ -74,11 +79,18 @@
         ('gp_correction', GP_CORRECTION_REGISTRATION_APPLICATION),
     ]
 )
-def test_valid_firms_correction(session, test_name, filing):
+def test_valid_firms_correction(monkeypatch, app, session, jwt, test_name, filing):
     """Test that a valid Firms correction passes validation."""
+    token = helper_create_jwt(jwt)
+    headers = {'Authorization': 'Bearer ' + token}
+
+    def mock_auth(one, two):  # pylint: disable=unused-argument; mocks of library methods
+        return headers[one]
```

Comment:
i think maybe you can put this at the top of the test file and share this across all the tests using it.

Reply:
actually nvm